### ISSUE TITLE
Add support to release aarch64 wheels

### ISSUE THREAD

**Issue by odidev** [mattandahalfew/Levenshtein_search#24]
Problem
---------
On aarch64, ‘pip install Levenshtein-search’ builds the wheels from source code and then installs it. It requires the user to have a development environment installed on his system. Also, it takes some time to build the wheels than downloading and extracting the wheels from pypi. 

Resolution
-----------
On aarch64, ‘pip install Levenshtein-search’ should download the wheels from pypi 

@mattandahalfew Please let me know your interest in releasing aarch64 wheels. I can help in this. Is there any plan to move to Travis-ci.com? If not, could you please tell the steps/CI which you are using presently to release the wheel on pypi?

**Comment by mattandahalfew** [comment]
Yes, I'm interested in the aarch64 wheels. Your help would be appreciated.

**Comment by odidev** [comment]
@mattandahalfew Could you please tell the steps/CI which you are using presently to release the wheel on pypi?

**Comment by mattandahalfew** [comment]
PyPi has an issue with my uploading the Mac OSX aarch64 wheels:
HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
Binary wheel 'Levenshtein_search-1.4.5-cp38-cp38-macosx_10_14_6_arm64.whl' has an unsupported platform tag 'macosx_10_14_6_arm64'.

**Comment by mattandahalfew** [comment]
I see that travis-ci.org has moved to travis-ci.com. I will work moving there. Are you interested in the mac aarch64 wheels or just linux?

**Comment by odidev** [comment]
> Are you interested in the mac aarch64 wheels or just linux?

Thanks for your reply. I am interested in just linux aarch64 wheels.
